app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,14 @@
 <resources>
-    <string name="app_name">Pokedex</string>
+    <string name="app_name">PokeDex</string>
+    <string name="nav_button_home">Home</string>
+    <string name="nav_button_by_type">By Type</string>
+    <!-- TODO: Remove or change this placeholder text -->
```

Comment:
Remove commented code!

Reply:
Commented code was removed from strings.xml file, please allow merging

Reply:
Ok, merge to master